waku-bindings/src/node/config.rs

```diff
@@ -20,10 +20,54 @@ pub struct WakuNodeConfig {
     /// Secp256k1 private key in Hex format (`0x123...abc`). Default random
     #[serde(with = "secret_key_serde", rename = "key")]
     pub node_key: Option<SecretKey>,
+    /// Cluster id that the node is running in
+    #[default(Some(0))]
+    pub cluster_id: Option<usize>,
     /// Enable relay protocol. Default `true`
     #[default(Some(true))]
     pub relay: Option<bool>,
     pub relay_topics: Vec<String>,
+    /// RLN configuration
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub rln_relay: Option<RLNConfig>,
+}
+
+/// RLN Relay configuration
+#[derive(Clone, SmartDefault, Serialize, Deserialize, Debug)]
```

Comment:
Default value for RLN conf don't really make sense IMO. We may want to force the user to input a config instead by not impl. any default for it.